Composer/packages/server/src/services/featureFlags.ts

```diff
@@ -28,6 +28,9 @@ export class FeatureFlagService {
       FeatureFlagService.currentFeatureFlagMap[key] = {
         ...FeatureFlagService.defaultFeatureFlags[key],
         ...FeatureFlagService.currentFeatureFlagMap[key],
+        displayName: FeatureFlagService.defaultFeatureFlags[key].displayName,
+        description: FeatureFlagService.defaultFeatureFlags[key].description,
+        documentationLink: FeatureFlagService.defaultFeatureFlags[key].documentationLink,
```

Comment:
Is it true that the only value we want to persist is the enabled state? I think we can refactor this code to reflect that.

```ts
FeatureFlagService.currentFeatureFlagMap[key] = {
  ...FeatureFlagService.defaultFeatureFlags[key],
  enabled: FeatureFlagService.currentFeatureFlagMap[key].enabled ?? FeatureFlagService.defaultFeatureFlags[key].enabled
}
```

Reply:
You could do the above if you always saved to storage each time updateFeatureFlags is called which I believe is on app start. Even if the feature flags have not changed. The current code is a bit more robust but checks if something has in fact changed and only saves if it has. So we would lose that feature with the suggested refactor but would result in slightly less and more simplified code.